src/repositories/helpers/MessageBuilder.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-len */
+import { MessageActionRow, MessageButton, MessageEmbed } from "discord.js";
+import { BallChaser } from "../../types/common";
+
+export default class MessageBuilder {
+  
+  static readonly normIconURL = "https://raw.githubusercontent.com/mattwells19/UNCC-Six-Mans.js/main/media/norm_still.png";
+
+  static readonly buttons = new MessageActionRow()
```

Comment:
Yeah - I found this out while working on the next branch.  Changed here, as it was already changed it my next branch. Since this is the message builder I left here since the buttons are part of the message, and just change to reflect the action the buttons are called for.